src/applications/claims-status/components/ClaimDetailLayout.jsx

```diff
@@ -24,6 +24,16 @@ const getBreadcrumbText = (currentTab, claimType) => {
   return `${currentTab} ${joiner} your ${claimType} claim`;
 };
 
+const showAdditionalContentions = () => {
```

Comment:
I think we may be better off moving this into it's own component since there is a lot more logic than there was before. Also wondering if we should be using refs instead of `document.getElementByClassName` as that's sort of an antipattern in react